sdk/src/tx/whileValidTxSender.ts

```diff
@@ -139,7 +139,7 @@ export class WhileValidTxSender extends BaseTxSender {
 
 		// handle subclass-specific side effects
 		const txSig = bs58.encode(
-			signedTx.signatures[0]?.signature || signedTx.signatures[0]
+			signedTx.signature || signedTx.signatures[0]?.signature
```

Comment:
this isn't relevant to these changes it just prevented me building the SDK locally - something weird that came up recently that's fixed in master